main.cpp

```diff
@@ -136,27 +195,28 @@ int main()
     auto carRented = rentACar(6, 2); 
     
     //1)
-    
+    auto answer = answerToEverything(false, 1.79f, 42);
     //2)
-    
+    auto currencyValue = convertCurrency('d', 'e', 87.99f, true);
     //3)
-    
+    delayLoudspeakers(200, true, 24.3f);
     //4)
-    
+    auto connectMic = needsPhantomPower('d');
     //5)
-    
+    auto roomSize = getSurfaceArea(200, 11);
     //6)
-    
+    waveLength(0.78f, 440.0f, 343.6);
     //7)
-    
+    print(4, false, true);
     //8)
-    
+    auto helicopterSpecs = thrustHelicopter(7.3f, 5.2f, 3);
     //9)
-    
+    vrTrackingSensor('r', true, false);
     //10)
+    auto newVolume = amplifySignal(-8, 1.5f);
     
-    
-    ignoreUnused(carRented);
+    ignoreUnused(carRented);
+    ignoreUnused(answer, currencyValue,connectMic, roomSize, helicopterSpecs, newVolume);

```

Comment:
cal `ignoreUnused` once, 
pass every variable in `main()` to it.
be consistent with the space after every `,`
- [ ] revise

Reply:
Ok, just to be sure: I only pass the local variables to ignoreUnused(), right? Functions of return type void I only call them within `main()`  but I don't pass them to `ignoreUnused`.